package.json

```diff
@@ -1,36 +1,24 @@
 {
-  "name": "create-dao",
-  "version": "1.4.3",
-  "private": "true",
-  "description": "",
+  "name": "root",
```

Comment:
I think it's fairly typical for the root `package.json` to have a `name` that matches the repo name. Not a big deal but just wanted to throw that out there!

Reply:
The package.json file in the create-dao package has the "create-dao" name already.